documentation/directory-structure.md

```diff
@@ -0,0 +1,247 @@
+
+# ----- <dataPlaneSubfolder>: Microsoft.ConfidentialLedger
+
+/data-plane/Microsoft.ConfidentialLedger/stable
+/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13
+/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/examples
+
+/data-plane/Microsoft.ConfidentialLedger/preview
+/data-plane/Microsoft.ConfidentialLedger/preview/2023-01-18-preview
+/data-plane/Microsoft.ConfidentialLedger/preview/2023-01-18-preview/examples
+# ... more previews here
+
+# ----- <dataPlaneSubfolder>: Microsoft.ManagedCcf
+
+/data-plane/Microsoft.ManagedCcf/preview/2023-06-01-preview
+/data-plane/Microsoft.ManagedCcf/preview/2023-06-01-preview/examples
+
+# ===== resource-manager
+
+/resource-manager/README.md
+
+# ----- resource-manager RP Namespace (<RPNS>): Microsoft.ConfidentialLedger
+
+/resource-manager/Microsoft.ConfidentialLedger/stable/2022-05-13
+/resource-manager/Microsoft.ConfidentialLedger/stable/2022-05-13/examples
+
+/resource-manager/Microsoft.ConfidentialLedger/preview/2023-06-28-preview
+/resource-manager/Microsoft.ConfidentialLedger/preview/2023-06-28-preview/examples
+# ... more previews here
+/resource-manager/Microsoft.ConfidentialLedger/preview/2020-12-01-preview
+/resource-manager/Microsoft.ConfidentialLedger/preview/2020-12-01-preview/examples
+
+```
+
+For another example, see [`specification/eventgrid`].
+
+## Naming guidelines for `specification` folder contents
+
+- Folder names should be singular (e.g. `keyvault` not `keyvaults` ) -- this removes ambiguity for some non-english speakers.
+- Generic folder names should be lower-case.
+- Resource Provider Namespace (`<RPNS>`) folders can be PascalCased (e.g. `KeyVault`).
+- For file names, any casing is allowed.
+- When in doubt, mimic naming of the examples provided in this article.
+
+## `specification/common-types`
+
+The special directory of [`specification/common-types`] contains shared definitions that can be reused across all Azure team services in their
+`specification` child folders.
+
+## Advanced scenario: service group
+
+In case of big Azure teams, their `specification/<azureTeam>` hosts multiple services, together known as `service group`.
+The main difference between one service and a service group is how they are presented to Azure customers:
+One service has one SDK package and one documentation portal, while a service group has separate SDK package for each service and separate documentation.
+
+For example, [`specification/containerservice`] is a `service group` for both `aks` and `fleet` services.
+
+The doc for `aks` is [Azure Kubernetes Service]. It points to aks REST reference e.g. for [API version `2024-01-01`][aks REST reference 2024-01-01],
+which corresponds to [`specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2024-01-01`].
+
+The doc for `fleet` is [Azure Kubernetes Fleet Manager]. It point to fleet REST reference, e.g. for [API version `2023-10-15`][fleet REST reference 2023-10-15],
```

Comment:
>`fleet` is [Azure Kubernetes Fleet Manager] [](http://example.com/codeflow?start=12&length=43)

same here, a link will be nice

Reply:
It is indeed hyperlinked. I am using markdown reference links. It links to https://learn.microsoft.com/en-us/azure/kubernetes-fleet/